addons/disco-like-workflows/@{{ cookiecutter.project_slug }}/.github/workflows/cleanup-feature-branch.yaml

```diff
@@ -0,0 +1,35 @@
+name: Cleanup Feature Branch
+
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
Because of the possibility to deploy a feature branch without a PR (workflow_dispatch), the condition should be "on: delete". cleanup-feature-branch.yaml file to replace with actual cleanup script in disco (https://github.com/DND-IT/disco/blob/master/.github/workflows/cleanup-feature-branch.yml)

Reply:
@rbapst-tamedia added the dispatch wf condition.

Reply:
@rbapst-tamedia at the end I've chosen to make the template as generic as possible, therefore a setting specific for Disco will be applied in the PR created by the template